app/models/application.rb

```diff
@@ -0,0 +1,9 @@
+class Application < ApplicationRecord
+  validates :phone_number, :street, :city, :state, :zip_code, :country, :birth_place, :birth_date,           :country_of_citizenship, :occupation, presence: true
```

Comment:
Lets add a break between these symbolized keys, so like something like this:
`validates :phone_number, :street, :city, :state, :zip_code, 
  :country, :birth_place, :birth_date,           
  :country_of_citizenship, :occupation, presence: true`